src/mixins/date-mixin.ts

```diff
@@ -75,13 +75,13 @@ export default class DateMixin extends Vue {
    * @example "2021-01-01 07:00:00 GMT" -> "Dec 31, 2020"
    * @example "2021-01-01 08:00:00 GMT" -> "Jan 1, 2021"
    */
-  dateToPacificDate (date: Date): string {
+  dateToPacificDate (date: Date, longMonth = false): string {
```

Comment:
This should be ported to the other UIs. Any takers for Create UI (and possibly BCRS Shared Components)?

Reply:
OK, I've got this as an item in my todo list ... I will get to it some time.